docs/7.x/cluster.md

```diff
@@ -388,13 +388,13 @@ updating app from 0.0.1-alpha.1 to 0.0.1-alpha.2
 Deploying agents on nodes
 The operation has been created in manual mode.
 
-See https://gravitational.com/gravity/docs/cluster/#managing-operations for details on working with operation plan.
+See https://gravitational.com/gravity/docs/cluster.md#managing-operations for details on working with operation plan.
```

Comment:
This seems unusual compared to other urls.  I believe mkdocs strips the `.md` before rendering to end users for other links?  Will that be done here too (being a fully qualified url instead of a relative url)?

Reply:
This should ideally be a relative path for the purposes of working with documentation locally.

Reply:
In my patch drop just now, I didn't address any fully qualified urls, as I was focusing on `milv -ignore-external` output (which will be what our CI runs on each PR.  I'm willing to forgo external link cleanup (including this) for the time being - as this PR is becoming a bit of a slog, and I'd love to get it in.  I have some other work (for CI, to make sure we don't regress) based on top of it.